modules/core/src/main/scalanative/ChangeModeNative.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 Neandertech
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cue4s
+
+import scala.scalanative.meta.LinktimeInfo
+
+trait ChangeModeNative:
+  def changeMode(rawMode: Boolean): Boolean
+  def getchar(): Int
+
+object ChangeModeNative:
```

Comment:
Somehow calling this `CallingMode` breaks the linkage... Maybe because there's a `CallingMode.java` in the JVM definitions messes something up 🤷

Reply:
> I wonder if that's why you called the previous file Changemode 🤔

That was an accident actually, but I guess a happy one..

Reply:
The java files being added to the native build definitely can mess up things, I need to fix the build somehow